tests/js/spec/views/accountNotifications.spec.jsx

```diff
@@ -20,15 +19,10 @@ describe('AccountNotifications', function() {
         subscribeByDefault: true,
       },
     });
+  });
 
-    MockApiClient.addMockResponse({
```

Comment:
@billyvg Do you know why was this here? It's got the same URL as the first one so it would be ignored

Reply:
Probably just copy pasta leftovers